pkg/manifests/manifests.go

```diff
@@ -1769,6 +1790,19 @@ func (f *Factory) PrometheusAdapterDeployment(apiAuthSecretName string, requesth
 		},
 	)
 
+	enableAudit, err := auditLogsEnabled(config.AuditLevel)
```

Comment:
Even if the audit log level is set to `None`, we could have everything set up in prometheus-adapter to produce audit logs. I think that we should not introduce a logic saying that this particular level will produce or not logs and stick to set up everything by default and leave it up to the apiserver to handle the logs based on the policy level.